gantry/tests/test_linear_motion_config.cpp

```diff
@@ -7,8 +7,10 @@ TEST_CASE("Linear motion system using a belt") {
     GIVEN("OT2 X,Y gantry config") {
         struct LinearMotionSystemConfig<BeltConfig> linearConfig {
             .mech_config = BeltConfig{.pulley_diameter = 12.7},
-            .steps_per_rev = 200, .microstep = 32,
+            .steps_per_rev = 200, .microstep = 32, .encoder_ppr = 1000,
         };
         REQUIRE(linearConfig.get_steps_per_mm() == 160.40813f);
+        REQUIRE(linearConfig.get_encoder_pulses_per_mm() ==
+                100.25508f);  // 25.063770566
```

Comment:
confused - what's the number in the comment and why is it different?

Reply:
oh sorry, that was a note, bad habit of mine... That's when i was testing the first logic of the conversion and i was getting that number and i knew it was wrong.. I'll remove these comments...